src/ranged.cpp

```diff
@@ -169,9 +170,17 @@ bool player::handle_gun_damage( item &it )
     const cata::optional<islot_gun> &firing = it.type->gun;
     if( !it.has_flag( "NEVER_JAMS" ) &&
         x_in_y( dirt_dbl * dirt_dbl * dirt_dbl, 1000000000000.0 ) ) {
```

Comment:
Not introduced here, but this is a magic number, and should be explained somehow